src/tests/bitsets/test_stdlib_bitset_64.f90

```diff
@@ -0,0 +1,744 @@
+program test_stdlib_bitset_64
+    use, intrinsic :: iso_fortran_env, only : int8, int16, int32, int64
+    use stdlib_bitsets
+    character(*), parameter :: &
+        bitstring_0   = '000000000000000000000000000000000', &
+        bitstring_33  = '100000000000000000000000000000000', &
+        bitstring_all = '111111111111111111111111111111111'
+    type(bitset_64) :: set0, set1, set2, set3, set4, set5
+    integer         :: status
+    character(:), allocatable :: string0
+
+    call test_string_operations()
+
+    call test_io()
+
+    call test_initialization()
+
+    call test_bitset_inquiry()
+
+    call test_bit_operations()
+
+    call test_bitset_comparisons()
+
+    call test_bitset_operations()
+
+contains
+
+    subroutine test_string_operations()
+        character(*), parameter:: procedure = 'TEST_STRING_OPERATIONS'
+
+        write(*,*)
+        write(*,*) 'Test string operations: from_string, read_bitset, ' // &
+            'to_string, and write_bitset'
+
+        call set0 % from_string( bitstring_0 )
+        if ( bits(set0) /= 33 ) then
```

Comment:
Suggestion only (does not need to be applied if not wanted/too much work):
```suggestion
        call check(bits(set0) /= 33, msg =  error stop procedure // ' from_string failed to interpret ' // &
                'bitstring_0 size properly.' )
```

Reply:
Doing it everywhere seems like a lot of work, particularly for those cases with `if...then...else if...` cascades, and doing it for the few cases without the `if...then...else if...` cascades is liable to result in an inconsistent style of error reporting.

Reply:
I understand your comment. Good for me as it is.